CMakeLists.txt

```diff
@@ -152,8 +152,15 @@ set(HERMES_ENABLE_UNDEFINED_BEHAVIOR_SANITIZER OFF CACHE BOOL
 set(HERMES_ENABLE_THREAD_SANITIZER OFF CACHE BOOL
   "Enable -fsanitize=thread")
 
-set(HERMES_ENABLE_FUZZING OFF CACHE BOOL
-  "Enable fuzzing")
+# Enable Trace PC Guard
+set(HERMES_ENABLE_TRACE_PC_GUARD OFF CACHE BOOL
```

Comment:
What does `-fsanitize-coverage=trace-pc-guard` do? Should it be enabled for any fuzzing mode?
I would say just manage this flag with HERMES_ENABLE_FUZZILLI instead of making a separate variable for it. If libfuzzer needs it as well you can add the flag if either is defined.

Reply:
https://clang.llvm.org/docs/SanitizerCoverage.html It is for generating coverage information by inserting `__sanitizer_cov_trace_pc_guard` defined below to every edges. There are other use cases (coverage testing, etc.). The older version of Libfuzzer benefits from this flag. I feel it might be better to keep it as an independent flag.